library/src/morpheus-master/sass/modules/tool/mycalendar/_mycalendar.scss

```diff
@@ -0,0 +1,203 @@
+.#{$namespace}sakai-mycalendar {
+
+    li.mycalendar-event {
+        list-style-type: none;
+    }
+    .ui-datepicker-title {
+        font-weight: 500 !important;
+        font-size: 16px !important;
+        margin: 0 0 !important;
+    }
+    .mycalendar-event-heading {
+        display: flex;
+        justify-content: center;
+        align-items: middle;
+        color: #777;
+        border-top:1px solid lightgrey;
+        line-height: 24px;
+        padding-top: 20px;
+        padding-bottom: 20px;
+        font-weight: bold;
+    }
+    #mycalendar-event-heading-text {
+        font-weight: 500;
+        font-size: 16px;
+        margin-left: 15px;
+        margin-right: 15px;
+    }
+    .fa-chevron-circle-left:before, .fa-chevron-circle-right:before {
+        font-size: 24px;
+    }
+
+    .left-arrow, .right-arrow {
+        cursor: pointer;
+    }
+
+    .mycalendar-event {
+        margin-bottom: 15px;
+    }
+
+    .mycalendar-event-time {
+        width: 80px;
+    }
+
+    .mycalendar-event-type {
+        font-weight: bold;
+    }
+    .Mrphs-container.Mrphs-sakai-mycalendar {
```

Comment:
I believe all reference to "Mrphs" should be replaced with the namespace variable?